ext/openssl/ossl_pkcs7.c

```diff
@@ -878,6 +889,24 @@ ossl_pkcs7_to_pem(VALUE self)
     return str;
 }
 
+static VALUE
+ossl_pkcs7_finalize(VALUE self, VALUE data, VALUE flags)
```

Comment:
`PKCS7#add_data` seems to do the similar thing as `PKCS7_final(p7, in, PKCS7_BINARY)` already.